### ISSUE TITLE
org.freedesktop.Platform and modules branch 22.08 are end-of-life

### ISSUE THREAD

**Issue by yesseruser** [flathub/gg.guilded.Guilded#59]
Branch 22.08 of the following are end-of-life:
- org.freedesktop.Platform
- org.freedesktop.Platform.VAAPI.Intel
- org.freedesktop.Platform.GL.default
- org.freedesktop.Platform.GL.default branch 22.08-extra

A message saying this pops up every time I update Flatpak packages. I suggest upgrading to a newer version like 24.08, if possible.

**Comment by onurbbr** [comment]
I agree with this. It can be annoying to see EOL output every time I type "flatpak update".
